### DISCUSSION TITLE
Paste in cURL, get HttpClient call code written?

### DISCUSSION THREAD

**Discussion by StachuDotNet** [darklang/dark#4384] [Suggestions]
Is this a feature that could eventually be in Dark? if so, what would it look like?
This functionality could probably live within a package?

**Comment by pbiggar** [comment]
This sounds like a really cool feature. We do a similar thing when you paste, we parse it and turn it into a dict.

I don't really see the package side of this, maybe we can make it so that users can extend this sort of feature via packages, but that seems orthogonal to this issue.

**Reply by StachuDotNet** [reply]
> I don't really see the package side of this

Wrote up some thoughts here: https://github.com/darklang/dark/discussions/4385
